data/cp_ranges.json

```diff
@@ -0,0 +1,126 @@
+{
+  "3": {
```

Comment:
Can we either A) Get this reduced to an equation, or B) Get the rest of the Pokemon added too it?
Maybe it would make more sense as one of the Pokemon stats? "raid_min_cp" or so? Open to your thoughts on that.

Reply:
I was thinking of doing this by calculation from stats, either way its some additional work unless theres some nice pokemon library out there. 

Calculated from the base stats would be best I think